polaris-e2e/cypress/e2e/ui/happy-path.cy.ts

```diff
@@ -8,51 +8,51 @@ const {
   TARGET_SEARCH_TEXT,
   TARGET_CASE_ID,
   PRE_SEARCH_DELAY_MS,
-} = Cypress.env()
+} = Cypress.env();
 
 describe("Happy Path", () => {
   it("can view a case", () => {
-    cy.on("uncaught:exception", () => false)
+    cy.on("uncaught:exception", () => false);
 
-    cy.fullLogin()
+    cy.fullLogin();
 
-    cy.clearCaseTracker(TARGET_URN, TARGET_CASE_ID)
-    cy.visit("/polaris-ui")
+    cy.clearCaseTracker(TARGET_URN, TARGET_CASE_ID);
+    cy.visit("/polaris-ui");
 
-    cy.findByTestId("input-search-urn").type(`${TARGET_URN}{enter}`)
+    cy.findByTestId("input-search-urn").type(`${TARGET_URN}{enter}`);
 
     // open case details page
-    cy.findByTestId(`link-${TARGET_URN}`).click()
+    cy.findByTestId(`link-${TARGET_URN}`).click();
 
     // is our defendant correct
-    cy.findByTestId("txt-defendant-name").contains(TARGET_DEFENDANT_NAME)
+    cy.findByTestId("txt-defendant-name").contains(TARGET_DEFENDANT_NAME);
 
     // open our target document
-    cy.findByTestId("btn-accordion-open-close-all").click()
-    cy.findByText(TARGET_DOCUMENT_NAME).click()
+    cy.findByTestId("btn-accordion-open-close-all").click();
+    cy.findByText(TARGET_DOCUMENT_NAME).click();
 
     // is our target document correct - has expected fragment
-    cy.findByTestId("btn-open-pdf").contains(TARGET_DOCUMENT_NAME)
-    cy.get("span").contains(TARGET_DOCUMENT_TEXT_FRAGMENT)
+    // cy.findByTestId("btn-open-pdf").contains(TARGET_DOCUMENT_NAME)
```

Comment:
this is only relevant change here, rest is the linting correction. commenting out the btn-open-pdf test